src/components/TextInput/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { CustomtextField } from './styles';
+import { Icon } from '../Icon';
+import { TextFieldErrorLabel, TextFieldHint, TextFieldLabel } from './hintComponent';
+import { TextFieldProps } from '@mui/material';
+import { theme } from '../../mui-theme';
```

Comment:
pls do not import the theme like this. if you need it in your component then use the `useTheme` hook from material ui